include/wlr/util/box.h

```diff
@@ -1,15 +1,20 @@
 /*
- * This an unstable interface of wlroots. No guarantees are made regarding the
- * future consistency of this API.
+* This is a stable interface of wlroots. Future changes will be limited to:
+ *
+ * - New functions
+ * - New struct members
+ * - New enum members
+ *
+ * Note that wlroots does not make an ABI compatibility promise - in the future,
+ * the layout and size of structs used by wlroots may change, requiring code
+ * depending on this header to be recompiled (but not edited).
+ *
+ * Breaking changes are announced in the release notes and follow a 1-year
+ * deprecation schedule.
  */
-#ifndef WLR_USE_UNSTABLE
-#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
-#endif
-
-#ifndef WLR_TYPES_WLR_BOX_H
-#define WLR_TYPES_WLR_BOX_H
+#ifndef WLR__BOX_H
```

Comment:
This guard should be named `WLR_UTIL_BOX_H` based on the file path.

Reply:
Woops, messed up my vim replace motion.